johnsonandjohnson/Bodiless-JS
# Chore: github action workflow

### Changes:
- Add github actions to build, lint and test.

### Example Runs:
- https://github.com/awm086/Bodiless-JS/runs/1457298278
- https://github.com/awm086/Bodiless-JS/actions/runs/384589225

### Test Instructions: 
- Ensure Current master build and pass all tests.
- Merge into the main branch.
- Issue PR to the main branch.

### Related
Related #681
